src/backend/common/data/commands/friend_codes.py

```diff
@@ -36,17 +37,20 @@ async def handle(self, db_wrapper, s3_wrapper):
                     if row_fc == self.fc:
                         raise Problem("You are already using this FC", status=400)
                 fc_count = len(rows)
-                fc_limits = {"mk8dx": 1, "mkt": 1, "mkw": 4, "mk7": 1, "mk8": 1}
+                fc_limits: dict[FriendCodeType, int] = {"switch": 1, "mkt": 1, "mkw": 4, "3ds": 1, "nnid": 1}
```

Comment:
I think we should call it "wii" and "wiiu" instead of "mkw" and "nnid", just to keep it consistent. MKT being an exception is fine since those friend codes are only related to MKT and not used in other games.